node/src/components/linear_chain_sync.rs

```diff
@@ -302,12 +302,22 @@ impl<I: Clone + PartialEq + 'static> LinearChainSync<I> {
                     "Block execution result doesn't match received block."
                 );
                 if self.is_recent_block(&block) {
-                    info!("downloaded recent block. finished synchronization");
+                    info!(
+                        hash=?block.hash(),
+                        height=?block.header().height(),
+                        era=?block.header().era_id(),
```

Comment:
```suggestion
                        era=block.header().era_id().0,
```
(That's what we do elsewhere, so that you read the word "era" only once.)